common/metric_types.py

```
@@ -59,18 +59,29 @@ async def collect_metric(self) -> None:
         """Collects single WebSocket message."""
         websocket = None
 
-        try:
+        async def _collect_ws_data():
```

Ruff (0.12.2)

Missing return type annotation for private function `_collect_ws_data`

(ANN202)